integration-cli/docker_cli_info_test.go

```diff
@@ -164,3 +164,21 @@ func (s *DockerSuite) TestInfoDebug(c *check.C) {
 	c.Assert(out, checker.Contains, "EventsListeners")
 	c.Assert(out, checker.Contains, "Docker Root Dir")
 }
+
+func (s *DockerSuite) TestInsecureRegistries(c *check.C) {
+	testRequires(c, SameHostDaemon, DaemonIsLinux)
+
+	registryCIDR := "192.168.1.0/24"
+	registryHost := "insecurehost.com:5000"
+
+	d := NewDaemon(c)
+	err := d.Start(fmt.Sprintf("--insecure-registry=%s", registryCIDR), fmt.Sprintf("--insecure-registry=%s", registryHost))
```

Comment:
As above, this would be slightly cleaner without the `Sprintf`s.

`err := d.Start("--insecure-registry="+registryCIDR, "--insecure-registry="+registryHost)`